lectures/opt_savings_2.md

```diff
@@ -112,6 +112,8 @@ def create_consumption_model(R=1.01,                    # Gross interest rate
 Here's the right hand side of the Bellman equation:
 
 ```{code-cell} ipython3
+:tags: [hide-input]
```

Comment:
@jstac do you want this to be hidden or visible?

Reply:
Good catch!  All code should be visible in both of the two lectures.